wix-embedded-mysql/src/main/java/com/wix/mysql/ScriptResolver.java

```diff
@@ -0,0 +1,77 @@
+package com.wix.mysql;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Lists;
+import org.apache.commons.io.filefilter.WildcardFileFilter;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.String.format;
+
+/**
+ * Helper for locating schema init scripts in a classpath.
+ *
+ * @author viliusl
+ * @since 06/06/15
+ */
+public class ScriptResolver {
+
+    /**
+     * Locates a single classPathFile in a classpath, ex. 'db/init_schema.sql'
+     */
```

Comment:
This one you can refactor/change - I searched in guava for something I can use, but I could not find it. It has utils for Resource discovery, but they did not work for me in a way that would be beneficial.